store.go

```diff
@@ -45,6 +46,9 @@ func loadInitialPowerTable(ctx context.Context, ec ec.Backend, m *manifest.Manif
 	} else if pt, err := ec.GetPowerTable(ctx, ts.Key()); err != nil {
 		log.Debugw("failed to load the bootstrap power table for F3 from state", "error", err)
 	} else {
+		if ptCid, err := certs.MakePowerTableCID(pt); err == nil {
```

Comment:
If we're computing this, we might as well check it against `m.InitialPowerTable` if defined.

Reply:
Check added, and warn on mismatch